main.go

```diff
@@ -80,7 +80,7 @@ func main() {
 					err = fmt.Errorf("%s", r)
 				}
 			}()
-			m, err = parse.NewParser().Parse(*input, fs)
+			m, _, err = loader.LoadSyslModule("", *input, fs, log)
```

Comment:
You could just commit this, ya know

Reply:
hahaha okay, will do! Just waiting for module version support to be merged in, then I'll bump the version and just add this change